src/util/sss_ini.c

```diff
@@ -180,14 +179,29 @@ static int sss_ini_access_check(struct sss_ini *self)
         flags |= INI_ACCESS_CHECK_UID | INI_ACCESS_CHECK_GID;
     }
 
-    ret = ini_config_access_check(self->file,
```

Comment:
There is similar check in `sss_ini_add_snippets()` that should be kept consistent with checks of main `sssd.conf`